client/displayservers/Wayland/shell_libdecor.c

```diff
@@ -103,14 +107,19 @@ static void libdecorCallback(uint32_t events, void * opaque)
   libdecor_dispatch(wlWm.libdecor, 0);
 }
 
-bool waylandShellInit(const char * title, bool fullscreen, bool maximize, bool borderless)
+bool waylandShellInit(const char * title, bool fullscreen, bool maximize, bool borderless, bool resizable)
```

Comment:
The corresponding arguments needs to be added in `shell_xdg.c` as well, even if it's just a stub. In general, the client needs to compile both with `libdecor` support and without.

Reply:
Added